demo/checkbox-demo.tsx

```diff
@@ -0,0 +1,29 @@
+import React = require('react');
+import {CheckBox} from "../src/components/checkbox/checkbox";
+
+export const demoCheckBoxText: string = 'covfefe';
```

Comment:
The demo should be representative illustration of what the component does. The core of the documentation. So let's make it a bit more friendly. The check can have some reflection in other place, some effect, changing some text on the page or something like that. (Have a look at some component demo page/playbook).